requirements-test.txt

```diff
@@ -3,3 +3,4 @@ coveralls
 isort
 flake8
```

Comment:
we should remove isort and flake8

Reply:
is isort and flake8 installed without requirement-test? @nemesisdesign

Reply:
like this here: https://github.com/openwisp/django-freeradius/pull/213
But wait for the build on that PR to complete so we can see if it really works :grin: (it should.. but we never know 100% sure until we try)

Reply:
ok i will waiting until build success and when all done i'll squash my commits.